docs/source/configuration-setup.rst

```diff
@@ -8,7 +8,7 @@ You provide this information either as keyword arguments to the constructor:
 
 .. code-block:: python
 
-   cluster = PBSCluster(cores=36, memory='100GB', queue='regular', ...)
+   cluster = PBSCluster(cores=36, memory='100GB', shebang,"#!/usr/bin/env bash", queue='regular', ...)
```

Comment:
Is there a comma instead of a equal sign here?
At the end, I would prefer to avoid exposing kwargs that are more rarely used everywhere in the docs. In this particular example I feel this is not needed.

Reply:
```suggestion
   cluster = PBSCluster(cores=36, memory='100GB', shebang="#!/usr/bin/env bash", queue='regular', ...)
```
great catch!